tests/dummy/app/controllers/hero/index.js

```diff
@@ -1,14 +1,18 @@
 import Ember from 'ember';
 import Move from 'ember-animated/motions/move';
 import Scale from 'ember-animated/motions/scale';
+import { FadeIn } from 'ember-animated/motions/opacity';
 
 export function * transition() {
   this.insertedSprites.forEach(sprite => {
```

Comment:
@ef4 I'm wondering if I can use `this.removedSprites` in `animated-bind`. I tried to add `FadeOut` effect on `removedSprites` but it doesn't work.

Reply:
`animated-bind` does support removedSprites, just like the other animators (`animated-each`, `animated-if`). But that is only directly useful if the `animated-bind` itself is not being destroyed. So in this example:

```hbs
{{#if showAge}}
  {{#animated-bind person.age use=someTransition as |age|}}
    <span>{{age}}</span>
  {{/animated-bind}}
{{/showAge}}
```
If you change `person.age`, the `someTransition` will see `removedSprites` with the old age span and `insertedSprites` with the new age span.

If you change `showAge` to false, the `someTransition` never has a chance to run. But we do measure the geometry of its sprites and retain references to their DOM elements, and hand them off to `farMatch` on the motion service. This allows any *other* animators that are about to animate to try to match the bound values (in this case, the value of `age`) and `startAt` the removed sprites.

So there are two solutions for how to actually make the old content fade away. One is to make sure the whole animation is being controlled by a component that isn't being destroyed. In the hero demo, it would be an `animated-outlet` (which is not implemented yet, but is basically just an animated-bind where the bound value is Ember's outlet state).

The other solution is to give people a way to use "ghost" sprites. We already retain copies of the DOM elements that were removed. The idea would be to truly move those elements into the new animator so they are fully useful as `removedSprites` even though they came from another already destroyed animator.

I think both solutions are good to have. The first solution is how liquid-fire has already always worked, the ghost sprite solution with far matching is an enhancement beyond that.